src/javascripts/components/analytics.mjs

```diff
@@ -16,6 +16,6 @@ export default function loadAnalytics () {
       j.async = true
       j.src = `https://www.googletagmanager.com/gtm.js?id=${i}${dl}`
       document.head.appendChild(j)
-    })(window, document, 'script', 'dataLayer', 'GTM-53XG2JT')
+    })(window, document, /** @type {const} */ ('script'), 'dataLayer', 'GTM-53XG2JT')
```

Comment:
As this isn't really 'our' script, would it make more sense to turn off all type checking for the snippet?

Reply:
I'm happy with that. Added `// @ts-nocheck` like we did for polyfills in the past